python/SWESimulators/KP07.py

```diff
@@ -41,7 +41,10 @@ def __init__(self, \
                  dx, dy, \
                  g, f=0.0, r=0.0, \
                  t=0.0, \
-                 theta=1.3, use_rk2=True, coriolis_beta=0.0, \
+                 dt=None, \
+                 dt_scale=1.0, \
```

Comment:
Why does this appear twice in the parameters list?

Reply:
That would be a mistake.